src/diffusers/schedulers/scheduling_dpmsolver_singlestep.py

```diff
@@ -189,6 +190,11 @@ def __init__(
             else:
                 raise NotImplementedError(f"{solver_type} does is not implemented for {self.__class__}")
 
+        if algorithm_type != "dpmsolver++" and final_sigmas_type == "denoise_to_zero":
```

Comment:
It would be nice to support `sde-dpmsolver++` in `DPMSolverSinglestepScheduler`. Is one of the best combinations in k-diffusers, and is missing in `diffusers`

Reply:
Yes, I agree. **+1** for `sde-dpmsolver++`